deep_translator/google_trans.py

```diff
@@ -51,6 +62,17 @@ def get_supported_languages(as_dict=False, **kwargs):
         """
         return GoogleTranslator.supported_languages if not as_dict else GoogleTranslator._languages
 
+    def is_secondary(self, lang):
+        """
+        Function to check if lang is a secondary name of any primary language
+        @param lang: language name
+        @return: primary name of a language if found otherwise 0
+        """
+        for primary_name, secondary_names in GOOGLE_LANGUAGES_SECONDARY_NAMES.items():
+            if (lang in secondary_names):
+                return primary_name
+        return 0
```

Comment:
you can return False instead of 0. It is more readable

Reply:
done